### PR TITLE
Hooks reorg

### PR DESCRIPTION
**Pull request by ciaranschutte**
- use barrel file
- clean up imports

### CODE REVIEW

File: src/app/hooks/index.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	useGlobalLoader,
+} from './GlobalLoaderProvider';
+export { ARGOTheme, default as ThemeProvider } from './ThemeProvider';
+export { default as ToastProvider, useToaster } from './ToastProvider';
+export * from './useApolloQuery';
+export { default as useCommonToasters } from './useCommonToasters';
+export { default as usePageContext } from './usePageContext';
+export * from './useSubmissionSystemStatus';
+export * from './useTimeout';
+export { default as useURLQueryState } from './useURLQueryState';
```
Comment (demariadaniel) [inline src/app/hooks/index.ts:35]: Looks like this is only imported in one place and then aliased again as `useUrlQueryState` (lower case)
May as well remove the double `as` and just export once as `useUrlQueryState`

**Comment by ciaranschutte** [comment]
@demariadaniel fixed thank you

File: src/app/components/Page.tsx [hunk 2]
```diff
@@ -18,11 +18,11 @@
  */
 'use client';
 
-import { useAuthContext } from '@/app/hooks/AuthProvider';
+import { useAuthContext } from '@/app/hooks';
 import { EmotionJSX } from '@emotion/react/types/jsx-namespace';
 import { DnaLoader, css } from '@icgc-argo/uikit';
 import { redirect } from 'next/navigation';
-import useUserRole, { UserRoleList } from '../hooks/useUserRole';
+import useUserRole, { UserRoleList } from '../hooks';
```
Comment (demariadaniel) [inline src/app/components/Page.tsx:25]: Did a manual review, this import is showing an error for me

Comment (ciaranschutte) [reply src/app/components/Page.tsx]: nice catch - fixed

**Review by demariadaniel — APPROVED** [review]

**Comment by demariadaniel** [comment]
Great cleanup!

**Pull request merged**